mxcube3/routes/qutils.py

```diff
@@ -359,6 +359,23 @@ def set_dc_params(model, entry, task_data):
                 _cpos.index = int(params['point'])
                 acq.acquisition_parameters.centred_position = _cpos
 
+    if params["helical"]:
```

Comment:
Hm, this was is already added by Mikels PR #283 are you sure that there is not a problem with the PRs ?